### DISCUSSION TITLE
Welcome to Stinsen Discussions!

### DISCUSSION THREAD

**Discussion by rundfunk47** [rundfunk47/stinsen#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by TsRebornz** [comment]
@rundfunk47  Is there any analogue of `self.navigationController!.popToViewController(controller, animated: true)` in Stinsen?

**Comment by rundfunk47** [comment]
> @rundfunk47  Is there any analogue of `self.navigationController!.popToViewController(controller, animated: true)` in Stinsen?

Yes, `focusFirst` should accomplish what you need.
